tests/lib/test_user.py

```diff
@@ -181,7 +181,7 @@ def test_0030_move_out(self):
         test_mac = "12:11:11:11:11:11"
         traffic_group = user.TrafficGroup.q.first()
 
-        new_user = UserHelper.move_in(
+        new_user, plain_password = UserHelper.move_in(
```

Comment:
Just a note: if you don't actually use that variable, you can replace it with an underscore: `new_user, _ = UserHelper.move_in(…)`